### DISCUSSION TITLE
GitHub Octernships - FAQs 🚀

### DISCUSSION THREAD

**Discussion by arkodyuti** [community/community#49380] [GitHub Education]
### Who is eligible for GitHub Octernships?

To apply for the [Octernships](https://education.github.com/students/octernships) program, you must follow the requirements listed below:

- 18+ years old
- A verified student on GitHub Global Campus
- An active contributor on GitHub
- Enrolled in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia.

> ✨ _Pro-tip: A good [GitHub profile README ](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)can help you stand out of the crowd!_ 

### Why is student eligibility restricted to these 10 countries?

Octernships will be available initially in 10 countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia. This is based on partners, active users, and creating access to opportunities. We are working hard to expand the program globally. 🌎

### How do I know I am a student on GitHub Global Campus?

A verified student can head to [Global Campus](https://education.github.com/globalcampus/student) and see Global Campus dashboard like this. **👇**

![](https://i.imgur.com/9Frn9ki.png)

Suppose you're ineligible or have yet to apply as a student to Global Campus. In that case, you'll see the option to **Join Global Campus. 👇**

![](https://i.imgur.com/LGni2bz.png)
You can join the Global Campus by submitting proof of student in the application to get access to GitHub Octernships.

### How to become an active contributor on GitHub?

There are several ways to become an active contributor on GitHub. However, contributing to Open Source is one of the most popular and recommended method. You can [find an open-source project to contribute](https://opensource.guide/how-to-contribute/#finding-a-project-to-contribute-to) to or [start your own open-source project,  ](https://opensource.guide/starting-a-project/) 

Active contributions on GitHub monthly will allow you to unlock the feature Octernships on GitHub Global Campus.

### I can see Octernship opportunities but cannot apply to them (greyed out) What does it mean?

You are a verified student and hence can see the Octernship opportunities, you are unable to apply to them due to one or more following reasons:
* You are not an active contributor on GitHub

> ✨ _Tip: It is a good practice to contribute to open source regularly and [renew](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#expiration-and-renewals) your student status if it expires._

### How much time is available to apply to a GitHub Octernship project?

Each project has a different deadline. Therefore, you will have time until that deadline to apply to the respective project. The deadline for each project will be visible on the project card

![](https://i.imgur.com/gM4CHl6.png)

### How much is an Octern’s stipend?

The stipend varies for every Octernship project. However, all stipends are displayed on the Octernship dashboard from a minimum of $500/per month (USD).

### Is GitHub Octernship a remote opportunity?

Yes, all opportunities on GitHub Octernship are remote!  

### How many chances do I get to apply for GitHub Octernships?

You can submit up to **four Octernship applications** on a rolling basis throughout the year, as long as you remain a verified student.i.e., at any given time, you can have up to four ongoing opportunities applied.

You can apply for another opportunity after completing one.

### I have got multiple opportunities. Can I accept all of them?

You can only accept one Octernship offer at a time, and all the other opportunities you applied for will be automatically rejected. 

### I was eligible for Octernships previously but not anymore, why?

If you have been eligible for [Octernships](https://education.github.com/students/octernships) in the past but have paused or stopped activity on GitHub, your eligibility status may change. 

In order to be eligible again, you need to become an active contributor on GitHub again. You can achieve this by [contributing to open source!](http://opensource.guide/)

**Comment by RajGM** [comment]
Hi @arkodyuti 

I am 18+ years old, a verified student on GitHub Global Campus, and an active contributor on GitHub.
And
I am enrolled in an institution and an Indian citizen by birth.
But it says, "I do not meet the criteria."

https://github.com/RajGM

![approved](https://user-images.githubusercontent.com/16620150/223325788-8599c194-90a5-4c89-80bc-bc50e6aa20fc.jpg)

![Octernship](https://user-images.githubusercontent.com/16620150/223325668-cac974ef-8fc1-48e6-a619-8f79529b0ce2.jpg)

**Reply by divais** [reply]
@RajGM You need to be enrolled in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia. More [here](https://education.github.com/students/octernships#faq)

**Reply by RajGM** [reply]
Hi @divais 
I have used the UoL email for a discount instead of the KIET one, causing the problem.

**Reply by RajGM** [reply]
Is there someway to fix this?

**Reply by samus-aran** [reply]
Hi @RajGM 

As @divais has stated, you need to be **currently** enrolled in an institution with work eligibility in the list of countries currently available for Octernships. You are not eligible if your student status of the Octernship institutions has expired.

**Comment by arpanaditya** [comment]


**Reply by divais** [reply]
Octernships would be available throughout the year. There are several ways to become an active contributor on GitHub. You can [find an open-source project to contribute](https://opensource.guide/how-to-contribute/#finding-a-project-to-contribute-to) to or [start your own open-source project,](https://opensource.guide/starting-a-project/) or [join community exchange](https://docs.github.com/en/education/contribute-with-github-community-exchange/getting-started-with-github-community-exchange). You can begin now and check https://education.github.com/globalcampus/octernships regularly, once you are eligible you can apply. Best wishes!! 🎉

**Reply by thormiwa** [reply]
@divais I have contributed in some outreachy repositories but i am still not eligible

**Reply by jai10-dev** [reply]
What is considered an active contributor(monthly)?

**Comment by Apprentice76** [comment]
Hi, 
First of all, thanks for giving us this awesome opportunity👏. I just have one question. I satisfy 3/4 of the requirements but my activities at GH have been subpar. How would the students be filtered? Am I automatically ineligible or can I begin now? Unfortunately, this is my final year, and I cannot apply next year :(. Was hoping to be a part of this.

**Reply by divais** [reply]
Octernships would be available throughout the year so you can begin now and check https://education.github.com/globalcampus/octernships. Once you are eligible you can apply. Best wishes!!

**Reply by heissanjay** [reply]
@Apprentice76 I'm in the same situation as yours :(

**Comment by memory-hunter** [comment]
Can you please make Github Octernships available in the region of Georgia(country)?

**Reply by samus-aran** [reply]
Hi @memory-hunter 👋 

Thank you for this request. We will gradually expand to more regions over time so do stay tune.

**Comment by amajaying** [comment]
After I apply, am I supposed to start working on project or should I wait for approval?

**Reply by R0h1t3** [reply]
Hey, where are the projects to apply for? I qualify for all the categories.

**Reply by divais** [reply]
You can start working on the project, the partner company would start evaluations at their end after / close to the apply by date.

**Reply by R0h1t3** [reply]
So it is like, I will directly start working on their open source project and give my contributions to them. Then they will have a look at it and decide whether to take me or not. Right???

**Reply by amajaying** [reply]
@divais So, is it like one can start working after clicking on "I'm interested" button? or should one wait for confirmation from the partner company?

**Reply by github-education** [reply]
After applying, you will receive a confirmation with the assignment information. Note the assignment is not the Octernship project itself, it's an initial step to allow our partners to evaluate your skills. After our partners review your assignment delivery, they will provide feedback and accept or reject it. If your assignment is accepted, you can begin working on the Octernship project. If your assignment is rejected, please don't worry, you are welcome to apply for other Octernship projects.

**Reply by amajaying** [reply]
Ah alright, the task listed on the 'Octernship' page is the assignment that I have to submit before the given deadline ?

If there's written "Submit by May 15, 2023" then, I will submit this assignment by that date and they'll verify and send the result and then only the actual Octernship starts, right?

**Reply by Smartking1** [reply]
How do I get to submit the assignment?

**Reply by samus-aran** [reply]
Hi @amajaying & @Smartking1 👋 

As @github-education explains that once the partner has reviewed and accepted your proposal in the form of the submitted assignment then will the Octernship commence.

We will provide more information to help guide our students through this process. Thank you for asking these questions 👍

**Comment by PrathakGarg** [comment]
On opening the octernship link, it says coming soon for me. And https://education.github.com/globalcampus/octernships redirects me to my global campus home page. Where do I see the available octernships?
![image](https://user-images.githubusercontent.com/75690201/223348185-c852a968-e958-4095-ab4a-8a041db80157.png)

**Reply by divais** [reply]
These changes will be gradually rolling out over the next few days.

**Reply by ashwinexe** [reply]
I suggest to check back to this [Octernships](https://education.github.com/globalcampus/octernships) page regularly

**Reply by DevYoma** [reply]
I am having the same issue too, it's showing me "Coming soon"

**Reply by imanishbarnwal** [reply]
Same for me, showing `Coming Soon`

**Comment by Pranav-yadav** [comment]
Hi @arkodyuti
Edit: cc: @divais 

I'm [Pranav](https://github.com/Pranav-yadav), an active open source contributor ([facebook/react-native](https://github.com/facebook/react-native)) as well as part of GitHub global campus.

When I visit the Octernship page it says coming soon. Does it mean I'm not eligible or octernships are _not_ yet available in the India region?

**Reply by divais** [reply]
These changes will be gradually rolling out over the next few days.

**Reply by Pranav-yadav** [reply]
Thanks for the info, @divais ma'am.
Looking forward to it.

PS: Is there any way we'll get to know when changes are rolled out or do we just need to _visit the page_ regularly :) ?

**Reply by samus-aran** [reply]
Hi @Pranav-yadav 👋 What a great question. Depending on the size of the changes and frequency of them we may provide periodic updates but we'll review this so we take in consideration of our students who are keen to apply. 👍

**Comment by rishipurwar1** [comment]
Hello @arkodyuti & @divais,

So far, I have made 152 contributions in 2023, and in 2022, I made 416 contributions. However, despite these contributions, I am still receiving the message, "Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship". 

I would like to know the minimum number of contributions we need to make per month/year to become eligible for this program.

**Reply by Tes-program** [reply]
Same here

**Comment by PradyumnaKrishna** [comment]
Just asking, there is limit of 4 applications. When does it reset If I apply to all 4 this month?

**Reply by imselmon** [reply]
You can submit up to four Octernship applications on a rolling basis throughout the year, as long as you remain a verified student.i.e., at any given time, you can have up to four ongoing opportunities applied.

**Comment by R0h1t3** [comment]
@codersalman @divais When I visit the Octernship page it says coming soon. So should wait for it to be open or am I not eligible?

**Reply by imselmon** [reply]
This will chnage in next few days

**Comment by Shrit1401** [comment]
Hey, I am Shrit I have doubt regarding how the sustem works, when I click 'I am Intreasted Button', after when the work is done will I get the stipend amount after it will be verified or will they give me more work, and also only one person would be selected or everyone who have applied

Thank You!

**Reply by samus-aran** [reply]
Hi @Shrit1401 👋 

As @github-education explains in an [earlier comment](https://github.com/orgs/community/discussions/49380#discussioncomment-5224042) that once the partner has reviewed and accepted your proposal in the form of the submitted assignment then will the Octernship commence.

We will provide more information to help guide our students through this process. Thank you for asking these questions 👍

**Comment by neelimagoogly** [comment]
I would like to know the minimum number of contributions we need to make per month/year to become eligible for this program?

**Comment by heysujal** [comment]
What are the chances of selection? I mean how many students can get selected for a single organization?
Like we have a limit to how many students get selected in programs like GSoC or mentorship programs.

**Reply by samus-aran** [reply]
Hi @heysujal 👋 

These are great questions. We'll update our FAQs and documentation to reflect this and provide more guidance for students applying.

**Comment by AnarchistHoneybun** [comment]
I'm from India, but it shows "coming soon" on Octernships page. Has it not been enabled yet or is it a problem on my end?

**Reply by amarachiugwu** [reply]
These changes will be gradually rolled out over the next few days.

**Comment by ACSG-64** [comment]
Great opportunity! however I would like to know if I am eligible: I am from and live in Colombia and can legally work in my country, however I am currently studying an undergraduate degree program at a British university in the distance learning modality. 
Since my university is not located in one of the 10 countries where the Octernships program is available, could this affect my eligibility? 
Thank you in advance for your answer.

**Reply by samus-aran** [reply]
Hi @ACSG-64 👋 

You need to be currently enrolled in an institution with work eligibility in the list of countries currently available for Octernships. This would unfortunately affect your eligibility. But thank you for raising this for other students who have the same circumstances seeking answers.

**Comment by pieterbrandsen** [comment]
I am 18 years old, was an student on github and am not in the list of countries yet.

Is there any chance non students can also apply. I would really like the challange.

**Comment by CodeWithShreyans** [comment]
I satisfy all the other requirements but am 16 years old.
Is it possible to get an exception or will the program expand to younger ages later?

**Reply by samus-aran** [reply]
Hi @DestroyerXyz 👋 

Thank you for your question but the current eligible age is 18 and older.

**Comment by chmodshubham** [comment]
is there any boundation to only contribute to our local expert GitHub repos for eligibility in the octernships or flexible to contribute to any projects? 
thanks in advance!

**Reply by divais** [reply]
You can contribute to any project on GitHub. 
Enjoy coding!

**Comment by NirmalScaria** [comment]
Heyy @arkodyuti ,  @divais , @amarachiugwu 
I have completed 464 contributions last year and own and manage a repo with 600+ stars... And when I try to apply for octernship, it says "Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship.". Could you please give some idea about what is the qualifying benchmark for octernship?
Thank you.

**Comment by Chirag8023** [comment]
Can graduated students who are not into any institute currently, can they have access to octernships ?

**Reply by amarachiugwu** [reply]
No, You must be enrolled in an institution and also meet the below criteria.

18+ years old
A verified student on GitHub Global Campus
An active contributor on GitHub
Enrolled in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines,
Thailand, Mexico, Nigeria, and Colombia.

**Comment by alvinpanerio** [comment]


**Reply by amarachiugwu** [reply]
You can try rephrasing your question.

**Comment by hdadhich01** [comment]
When is the anticipated release date for the program in the United States?

**Comment by jai10-dev** [comment]
What qualifies as an active github contributor?

**Comment by samus-aran** [comment]
Thank you all for the awesome interest in GitHub Octernships! 🎉

You have all provided invaluable feedback by asking very informative questions. These comments will help us improve our FAQs and guidance to our students to make applying to the Octernship program a much smoother process. 

If your question has not been answered we would ask you to create a new discussion in the [GitHub Education Community](https://github.com/orgs/community/discussions/categories/github-education) so we can assist further. ⭐

Thank you again for all your participation and hope to see you around in the community 👋
